build/bin/make-coverage

```diff
@@ -0,0 +1,5 @@
+#!/bin/bash
+wget https://dl.google.com/linux/direct/google-chrome-stable_current_amd64.deb
```

Comment:
you have to download it every time? Dockerfile doesn't allow install?

Reply:
We use someone else's image to run this stuff. I suppose we could make our own, but I don't like the maintenance headache of maintaining these build only images.